srcpkgs/tlpui/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'tlpui'
+pkgname=tlpui
+version=1.5.0
+revision=1
+wrksrc="TLPUI-${pkgname}-${version}"
+makedepends="python-setuptools"
```

Comment:
This should be `hostmakedepends` instead.  That will fix the cross-build.

Reply:
It's my first time, thanks for the points!